File: 프로그래머스/1/42748. K번째수/K번째수.java
import java.util.Arrays;
class Solution {
    public int[] solution(int[] array, int[][] commands) {
        int[] answer = new int[commands.length];

        for (int i = 0; i < commands.length; i++) {
            int start = commands[i][0];
            int end = commands[i][1];
            int num = commands[i][2];

            int[] subArray = Arrays.copyOfRange(array, start-1, end);
            subArray = Arrays.stream(subArray).sorted().toArray();
            answer[i] = subArray[num-1];
        }
        return answer;
    }
}

File: 프로그래머스/1/42748. K번째수/K번째수.js
function solution(array, commands) {
    let answer = [];

    for (let i = 0; i < commands.length; i++) {
        let start = commands[i][0];
        let end = commands[i][1];

        let arr = array.slice(start - 1, end).sort((a, b) => a - b);

        answer.push(arr[commands[i][2]-1]);
    }

    return answer;
}